project/DockerSeedPlugin.scala

```diff
@@ -100,14 +92,15 @@ object DockerSeedPlugin extends AutoPlugin {
       }
 
       private[this] val dockerSeedParser: Parser[Seq[ParseResult]] =
-        (BaseImage | ScalaVersion | JavaVersion | PlayVersion | playSlickVersion | WithDefaults | SbtVersion | DockerRegistry).*
+        (BaseImage | ScalaVersion | JavaVersion | PlayVersion | playSlickVersion | WithDefaults | SbtVersion |
+          DockerRegistry)
+          .*
```

Comment:
The indenting on this line ended up a bit weird, in case you want to adjust it in some other fashion